server/serializer/confluence_cloud.go

```diff
@@ -54,63 +66,126 @@ type Comment struct {
+
+	case CommentCreatedEvent:
+		message := fmt.Sprintf(ConfluenceCommentCreatedMessage, e.GetUserDisplayNameForCommentEvents(), e.GetPageDisplayNameForCommentEvents(baseURL, true), e.GetSpaceDisplayNameForCommentEvents(baseURL, true))
+		text := ""
+		if strings.TrimSpace(e.Comment.Body.View.Value) != "" {
+			text = fmt.Sprintf("**%s wrote:**\n> %s\n\n", e.GetUserDisplayNameForCommentEvents(), strings.TrimSpace(e.Comment.Body.View.Value))
+			attachment = &model.SlackAttachment{
+				Fallback: message,
+				Pretext:  message,
+				Text:     fmt.Sprintf("%s\n\n[**View in Confluence**](%s)", text, e.Comment.Self),
+			}
+		} else {
+			post.Message = fmt.Sprintf(ConfluenceEmptyCommentCreatedMessage, e.GetUserDisplayNameForCommentEvents(), e.Comment.Self, e.GetPageDisplayNameForCommentEvents(baseURL, true), e.GetSpaceDisplayNameForCommentEvents(baseURL, true))
+		}
+
 	case CommentUpdatedEvent:
-		message = fmt.Sprintf(confluenceCloudCommentUpdateMessage, comment.Self, comment.Parent.Title, comment.Parent.Self)
-	case PageRemovedEvent:
-		message = fmt.Sprintf(confluenceCloudPageDeleteMessage, page.Title, page.SpaceKey)
-	case CommentRemovedEvent:
-		message = fmt.Sprintf(confluenceCloudCommentDeleteMessage, comment.Parent.Title, comment.Parent.Self)
+		message := fmt.Sprintf(ConfluenceCommentUpdatedMessage, e.GetUserDisplayNameForCommentEvents(), e.GetPageDisplayNameForCommentEvents(baseURL, true), e.GetSpaceDisplayNameForCommentEvents(baseURL, true))
+		if strings.TrimSpace(e.Comment.Body.View.Value) != "" {
+			attachment = &model.SlackAttachment{
+				Fallback: message,
+				Pretext:  message,
+				Text:     fmt.Sprintf("**Updated Comment:**\n> %s\n\n[**View in Confluence**](%s)", strings.TrimSpace(e.Comment.Body.View.Value), e.Comment.Self),
+			}
+		} else {
+			post.Message = fmt.Sprintf(ConfluenceEmptyCommentUpdatedMessage, e.GetUserDisplayNameForCommentEvents(), e.Comment.Self, e.GetPageDisplayNameForCommentEvents(baseURL, true), e.GetSpaceDisplayNameForCommentEvents(baseURL, true))
+		}
+
+	case SpaceUpdatedEvent:
+		post.Message = fmt.Sprintf(ConfluenceSpaceUpdatedMessage, e.Space.SpaceKey, e.Space.Self)
 	default:
 		return nil
 	}
 
-	post := &model.Post{
-		UserId:  config.BotUserID,
-		Type:    model.POST_DEFAULT,
-		Message: message,
+	if attachment != nil {
+		model.ParseSlackAttachment(post, []*model.SlackAttachment{attachment})
 	}
 	return post
 }
 
 func (e ConfluenceCloudEvent) GetURL() string {
 	if e.Comment != nil {
 		return e.Comment.Self
-	} else if e.Page != nil {
+	}
+	if e.Page != nil {
 		return e.Page.Self
+	} else if e.Space != nil {
+		return e.Space.Self
+	}
+	if e.Space != nil {
+		return e.Space.Self
 	}
 	return ""
 }
 
 func (e ConfluenceCloudEvent) GetSpaceKey() string {
 	if e.Comment != nil {
 		return e.Comment.SpaceKey
-	} else if e.Page != nil {
+	}
+	if e.Page != nil {
 		return e.Page.SpaceKey
+	} else if e.Space != nil {
+		return e.Space.SpaceKey
+	}
+	if e.Space != nil {
```

Comment:
I don't think this if is needed, otherwise we need to refactor the previous one.